pyflagser/flagio.py

```diff
@@ -26,13 +26,13 @@ def loadflag(fname, fmt='csr', dtype=None):
     flag_matrix : matrix of format `fmt`
         Matrix representation of a directed/undirected weighted/unweighted
         graph. Diagonal elements are vertex weights. The way zero values are
-        handled depends on the format of the matrix. If the matrix is a dense
-        `np.ndarray`, all zeros are explicitly accounted for and denote
-        zero-weight edges, i.e., edges appeaing at filtration value zero.
-        If the matrix is a sparse `scipy.sparse` matrix, zeros on the diagonal
-        and off-diagonal zeros assigned directly are treated explicitly. Off-
-        diagonal zeros that have not been assigned directly are treated
-        implicitly, i.e., corresponds to an abscence of edge.
+        handled depends on `fmt`. If `fmt` is `'dense'`, all zeros are
```

Comment:
I think we need to use double ticks ( `` ) where we want monospace fonts, e.g. in the case of ``'dense'`` and ``scipy.sparse``.  Single ticks display as italics in the docs.